context/app/static/js/components/entity-tile/EntityTile/EntityTile.tsx

```diff
@@ -73,56 +73,25 @@ function ErrorTile({ entity_type, id }: Pick<EntityTileProps, 'id' | 'entity_typ
       })}
       variant="outlined"
     >
-      <Stack direction="row" gap={1} color="error.main" alignItems="center">
-        <StatusIcon status="error" sx={{ fontSize: '1.5rem' }} />
-        <Typography variant="subtitle1">{id}</Typography>
-        <IconButton onClick={() => copy(id)}>
-          <ContentCopyIcon color="info" />
-        </IconButton>
+      <Stack direction="row" gap={1}>
+        <StatusIcon status="error" sx={{ alignSelf: 'start', fontSize: '1.3rem' }} />
```

Comment:
Maybe `1.25rem`, I believe we use that more commonly throughout. I can't recall using `1.3rem` before, but maybe you have?